go.mod

```diff
@@ -100,6 +101,7 @@ require (
 
 replace (
 	cloud.google.com/go => cloud.google.com/go v0.100.2
+	github.com/argoproj-labs/argocd-operator => github.com/raghavi101/argocd-operator v0.0.0-20230918084530-babbca52187a
```

Comment:
Ensure that this remaping is removed once the changes for `argocd-operator` is merged.